Update subworkflow vag (#13)
vagkaratzas/proteinfamilies: docs/output.md

- `full_msa/`
  - `non_redundant/`
    - `mafft_align/`
      - `<samplename>/`
        - `<samplename>_*.fas`: fasta files with aligned amino acid sequences (after checking for sequence redundancy)

</details>

[mafft](https://github.com/GSLBiotech/mafft) is a fast but not very sensitive multiple sequence alignment tool.

> ### hmmer for updating families

Author: I will keep it as it is for consistency among blocks

<details markdown="1">
<summary>Output files</summary>

- `update_families/`
  - `hmmer/`
    - `concatenated/`
      - `<samplename>.hmm.gz`: (optional) concatenated compressed hmm model for all families in a given sample, to give as input to hmmsearch, to figure which families will be updated with new sequences
    - `hmmsearch/`
      - `<samplename>/`
        - `<samplename>.domtbl.gz`: (optional) hmmsearch results of input fasta file against existing families' HMMs
    - `hmmbuild/`
      - `<samplename>/`

>         - `<family_id>.hmm.gz`: (optional) compressed family HMM after update

Reviewer: ```suggestion
        - `<family_id>.hmm.gz`: (optional) compressed family HMM after the update
```

        - `<family_id>.hmmbuild.txt`: (optional) hmmbuild execution log
  - `branch_fasta/`
    - `hits/`

>       - `<family_id>.fasta`: (optional) subset of input fasta with hit sequences for each existing family

Reviewer: ```suggestion
      - `<family_id>.fasta`: (optional) subset of the input FASTA with hit sequences for each existing family
```

>     - `<samplename>.fasta.gz`: (optional) fasta file with all remaining non-hit input sequences, which will be passed to normal execution mode to create new families

Reviewer: ```suggestion
    - `<samplename>.fasta.gz`: (optional) FASTA file that contains all remaining non-hit input sequences, which will be passed to normal execution mode to create new families
```
